recipes/libarchive/all/patches/0003-3.7.5-cmake.patch

```diff
@@ -0,0 +1,84 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 582258c..b59a7e2 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -16,7 +16,7 @@ endif()
+ #
+ PROJECT(libarchive C)
+ #
+-SET(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/build/cmake")
++LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/build/cmake")
+ if(NOT CMAKE_RUNTIME_OUTPUT_DIRECTORY)
+   set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${libarchive_BINARY_DIR}/bin)
+ endif()
+@@ -469,7 +469,7 @@ ENDIF()
+ IF(ZLIB_FOUND)
+   SET(HAVE_LIBZ 1)
+   SET(HAVE_ZLIB_H 1)
+-  INCLUDE_DIRECTORIES(${ZLIB_INCLUDE_DIR})
++  LIST(APPEND ADDITIONAL_LIBS ZLIB::ZLIB)
+   LIST(APPEND ADDITIONAL_LIBS ${ZLIB_LIBRARIES})
```

Comment:
The previous patch was like 
```suggestion
  INCLUDE_DIRECTORIES(${ZLIB_INCLUDE_DIR})
-  LIST(APPEND ADDITIONAL_LIBS ${ZLIB_LIBRARIES})
+  LIST(APPEND ADDITIONAL_LIBS ZLIB::ZLIB)

```

Reply:
@ErniGH 
Thank you for your pointing out!
I fixed this.